packages/runtime-common/query.ts

```diff
@@ -4,14 +4,16 @@ import { assertJSONValue, assertJSONPrimitive } from './json-validation';
 import qs from 'qs';
 import { type CodeRef, isCodeRef } from './index';
 
+export type Page = {
+  number: number; // page.number is 0-based
+  size: number;
+  realmVersion?: number;
+}
```

Comment:
i don't think this will be necessary since the paging should be an interior detail of the `cardsThatReference()` method